ios/CozyReactNative/CozyReactNative.entitlements

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>aps-environment</key>
+	<string>development</string>
```

Comment:
CI should be updated to know to change to production if we're making a release?

Reply:
I need to check

Reply:
See https://github.com/cozy/cozy-flagship-app/pull/594/commits/94870f87f817fe593704fb38fe2c9196638459a6

No changes on iOS